openapi/index.yaml

```diff
@@ -25,6 +25,11 @@ components:
           example: []
           items:
             $ref: "#/components/schemas/Error"
+    FiscalCodeType:
```

Comment:
Why not just FiscalCode without **Type**?

Reply:
To be really compliant to interoperability standard  the italian "codice fiscale" must be named TaxCode as referred here

https://github.com/italia/form-pa/blob/bc4da698dfa61d80f7435c283f0cb0e78b8aa099/public/definitions.yaml#L358

Reply:
Also remember that this file is specific for the CloudGaaP project and not for the products of the catalog (like for example the document sharing service).

Reply:
Ok, we are not going to reference the TaxCode from anything external to CloudGaaP (e.g. document-sharing-service).